FV3/atmos_model.F90

```diff
@@ -115,7 +115,8 @@ module atmos_model_mod
                               FV3GFS_IPD_checksum,                       &
                               FV3GFS_diag_register, FV3GFS_diag_output,  &
                               DIAG_SIZE, FV3GFS_restart_write_coarse,    &
-                              FV3GFS_diag_register_coarse
+                              FV3GFS_diag_register_coarse, FV3GFS_register_vulcan_diagnostics, &
```

Comment:
what distinguishes the "vulcan" diagnostics? they are diagnostics from the physics that we're outputting through the dycore diagnostics manager? I wonder if there is a more descriptive name we could use.

Reply:
I felt the same way.  I toyed with adjectives like "conventional" or "standard" but those felt awkward to me since from the perspective of the physics they are actually unusual.  I guess we could be verbose and use something like `FV3GFS_register_diag_manager_controlled_diagnostics`?